pandas/io/excel/_calamine.py

```diff
@@ -28,13 +30,11 @@
 
     from pandas._typing import (
         FilePath,
-        NaTType,
         ReadBuffer,
-        Scalar,
         StorageOptions,
     )
 
-_CellValue = Union[int, float, str, bool, time, date, datetime, timedelta]
+_CellValueT = Union[int, float, str, bool, time, date, datetime, timedelta]
```

Comment:
Why is this changing? I think we only use `T` suffix for TypeVar, not type aliases.